AndroidLibrary/README.md

```diff
@@ -0,0 +1,4 @@
+#Barcode scanner
+###Simple Usage
+In this app, I had used 
+####Zxing and Lottie library.  
```

Comment:
1. Add the purpose of using this library in your project. 
2. Attach a screenshot or Gif of the application

Reply:
All asked changes have been made.